src/sds_data_model/metadata.py

```diff
@@ -134,7 +138,7 @@ def _get_value(
     root_element: Element,
     xpath: Union[str, Iterable[str], Iterable[Iterable[str]]],
     namespaces: Dict[str, str],
-) -> str:
+) -> Union[str, None]:
```

Comment:
`Union[T, None]` (where T is a type) is so common that it has a special syntax: `Optional[T]`, so this should be:

```suggestion
) -> Optional[str]:
```